aea/connections/base.py

```diff
@@ -86,7 +88,6 @@ def __init__(
         assert (
             super().public_id == self.connection_id
         ), "Connection ids in configuration and class not matching."
-        self._loop: Optional[AbstractEventLoop] = None
```

Comment:
@marcofavorito is it safe to remove this? I think we are loosing an ability to configure the object; so I don't see why this is beneficial.

Reply:
keep it simple. i made checks that the current test base works well without this feature.
also, this loop was set via property access outside the connection, that's tricky a bit and may confuse. in some code, loop was set directly, in some not. anyway, all the time loop was set to the current one.


for the moment it looks redundant without objective cases.

Reply:
I agree with the change. It simplifies the APIs and almost surely `asyncio.get_event_loop()` will return the right one, either if the connection run by the multiplexer or as a stand-alone connection.

In fact, modern Python versions deprecated `loop` parameters in many `asyncio` APIs, and I believe the same arguments for removal apply in this case too.

Good readings:
https://stackoverflow.com/a/60315290/7150172
https://vorpus.org/blog/some-thoughts-on-asynchronous-api-design-in-a-post-asyncawait-world/#getting-in-touch-with-your-event-loop